MoveIt! Scene Objects are ignored or dissapear

After starting the Driver and Moveit I can plan paths with MoveIt and the URSimulator, but when I try to add a scene object as obstacle, MoveIt ignores the Object. When I attach the Object to the base_link it's getting deleted.
I use the example with following commands:
```
ros2 launch ur_bringup ur_control.launch.py ur_type:=ur10e robot_ip:=127.0.0.1 launch_rviz:=false
```
Then starting the URCap
```
ros2 launch ur_bringup ur_moveit.launch.py ur_type:=ur10e robot_ip:="127.0.0.1" launch_rviz:=true
```

I will show you some screenshots to show the problem:

![image](https://user-images.githubusercontent.com/44194180/136022114-0f53b4b0-eadd-4715-bdb3-0e333be804cf.png)


Attaching Problem:
Klicking ok and then "Plan" in the Planning Tab let the Box_0 dissappear.

![image](https://user-images.githubusercontent.com/44194180/136022252-5c6e9436-cdeb-4277-ac20-4898064909a8.png)


And here is the Log Output from ur_moveit.launch.py:

```
ur@NBK-18:~$ ros2 launch ur_bringup ur_moveit.launch.py ur_type:=ur10e robot_ip:=127.0.0.1 launch_rviz:=true
[INFO] [launch]: All log files can be found below /home/ur/.ros/log/2021-10-05-14-31-29-260094-NBK-18-11396
[INFO] [launch]: Default logging verbosity is set to INFO
[INFO] [move_group-1]: process started with pid [11402]
[INFO] [mongo_wrapper_ros.py-2]: process started with pid [11404]
[INFO] [rviz2-3]: process started with pid [11406]
[INFO] [static_transform_publisher-4]: process started with pid [11408]
[static_transform_publisher-4] [INFO] [1633437090.400657300] [static_transform_publisher]: Spinning until killed publishing transform from 'world' to 'base_link'
[move_group-1] [WARN] [1633437090.460087900] [move_group.move_group]: MoveGroup launched without ~default_planning_pipeline specifying the namespace for the default planning pipeline configuration
[move_group-1] [WARN] [1633437090.461019900] [move_group.move_group]: Falling back to using the the move_group node namespace (deprecated behavior).
[move_group-1] Parsing robot urdf xml string.
[move_group-1] [INFO] [1633437090.468158900] [moveit_rdf_loader.rdf_loader]: Loaded robot model in 0.0019955 seconds
[move_group-1] [INFO] [1633437090.468252300] [moveit_robot_model.robot_model]: Loading robot model 'ur'...
[rviz2-3] QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-ur'
[move_group-1] Link base had 0 children
[move_group-1] Link base_link_inertia had 1 children
[move_group-1] Link shoulder_link had 1 children
[move_group-1] Link upper_arm_link had 1 children
[move_group-1] Link forearm_link had 1 children
[move_group-1] Link wrist_1_link had 1 children
[move_group-1] Link wrist_2_link had 1 children
[move_group-1] Link wrist_3_link had 1 children
[move_group-1] Link flange had 1 children
[move_group-1] Link tool0 had 0 children
[move_group-1] [INFO] [1633437090.552503200] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Publishing maintained planning scene on 'monitored_planning_scene'
[move_group-1] [INFO] [1633437090.552735000] [moveit.ros_planning_interface.moveit_cpp]: Listening to '/joint_states' for joint states
[move_group-1] [INFO] [1633437090.554331800] [moveit_ros.current_state_monitor]: Listening to joint states on topic '/joint_states'
[move_group-1] [INFO] [1633437090.556117000] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Listening to '/move_group/planning_scene_monitor' for attached collision objects
[move_group-1] [INFO] [1633437090.556232000] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Starting planning scene monitor
[move_group-1] [INFO] [1633437090.556951200] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Listening to '/move_group/publish_planning_scene'
[move_group-1] [INFO] [1633437090.556979500] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Starting world geometry update monitor for collision objects, attached objects, octomap updates.
[move_group-1] [INFO] [1633437090.558818600] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Listening to 'collision_object'
[move_group-1] [INFO] [1633437090.560537500] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Listening to 'planning_scene_world' for planning scene world geometry
[move_group-1] [WARN] [1633437090.560665700] [moveit.ros.occupancy_map_monitor]: Resolution not specified for Octomap. Assuming resolution = 0.1 instead
[move_group-1] [ERROR] [1633437090.560710200] [moveit.ros.occupancy_map_monitor]: No 3D sensor plugin(s) defined for octomap updates
[move_group-1] [INFO] [1633437090.570554300] [moveit.ros_planning_interface.moveit_cpp]: Loading planning pipeline 'move_group'
[move_group-1] [INFO] [1633437090.612436400] [moveit.ros_planning.planning_pipeline]: Using planning interface 'OMPL'
[move_group-1] [INFO] [1633437090.619465200] [moveit_ros.add_time_optimal_parameterization]: Param 'move_group.path_tolerance' was not set. Using default value: 0.100000
[move_group-1] [INFO] [1633437090.619631100] [moveit_ros.add_time_optimal_parameterization]: Param 'move_group.resample_dt' was not set. Using default value: 0.100000
[move_group-1] [INFO] [1633437090.619643100] [moveit_ros.add_time_optimal_parameterization]: Param 'move_group.min_angle_change' was not set. Using default value: 0.001000
[move_group-1] [INFO] [1633437090.619697900] [moveit_ros.fix_workspace_bounds]: Param 'move_group.default_workspace_bounds' was not set. Using default value: 10.000000
[move_group-1] [INFO] [1633437090.619726900] [moveit_ros.fix_start_state_bounds]: Param 'move_group.start_state_max_bounds_error' was set to 0.050000
[move_group-1] [INFO] [1633437090.619735100] [moveit_ros.fix_start_state_bounds]: Param 'move_group.start_state_max_dt' was not set. Using default value: 0.500000
[move_group-1] [INFO] [1633437090.619753700] [moveit_ros.fix_start_state_collision]: Param 'move_group.start_state_max_dt' was not set. Using default value: 0.500000
[move_group-1] [INFO] [1633437090.619761300] [moveit_ros.fix_start_state_collision]: Param 'move_group.jiggle_fraction' was not set. Using default value: 0.020000
[move_group-1] [INFO] [1633437090.619788100] [moveit_ros.fix_start_state_collision]: Param 'move_group.max_sampling_attempts' was not set. Using default value: 100
[move_group-1] [INFO] [1633437090.619806900] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Add Time Optimal Parameterization'
[move_group-1] [INFO] [1633437090.619820300] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Fix Workspace Bounds'
[move_group-1] [INFO] [1633437090.619826200] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Fix Start State Bounds'
[move_group-1] [INFO] [1633437090.619831500] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Fix Start State In Collision'
[move_group-1] [INFO] [1633437090.619863200] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Fix Start State Path Constraints'
[move_group-1] [INFO] [1633437090.743778000] [moveit.plugins.moveit_simple_controller_manager]: Added FollowJointTrajectory controller for joint_trajectory_controller
[move_group-1] [INFO] [1633437090.744253200] [moveit.plugins.moveit_simple_controller_manager]: Returned 1 controllers in list
[move_group-1] [INFO] [1633437090.746581800] [moveit_ros.trajectory_execution_manager]: Trajectory execution is not managing controllers
[move_group-1] [INFO] [1633437090.746637000] [move_group.move_group]: MoveGroup debug mode is ON
[move_group-1] [INFO] [1633437090.788526000] [move_group.move_group]:
[move_group-1]
[move_group-1] ********************************************************
[move_group-1] * MoveGroup using:
[move_group-1] *     - ApplyPlanningSceneService
[move_group-1] *     - ClearOctomapService
[move_group-1] *     - CartesianPathService
[move_group-1] *     - ExecuteTrajectoryAction
[move_group-1] *     - GetPlanningSceneService
[move_group-1] *     - KinematicsService
[move_group-1] *     - MoveAction
[move_group-1] *     - MotionPlanService
[move_group-1] *     - QueryPlannersService
[move_group-1] *     - StateValidationService
[move_group-1] ********************************************************
[move_group-1]
[move_group-1] [INFO] [1633437090.788609900] [moveit_move_group_capabilities_base.move_group_context]: MoveGroup context using planning plugin ompl_interface/OMPLPlanner
[move_group-1] [INFO] [1633437090.788623600] [moveit_move_group_capabilities_base.move_group_context]: MoveGroup context initialization complete
[move_group-1] Loading 'move_group/ApplyPlanningSceneService'...
[move_group-1] Loading 'move_group/ClearOctomapService'...
[move_group-1] Loading 'move_group/MoveGroupCartesianPathService'...
[move_group-1] Loading 'move_group/MoveGroupExecuteTrajectoryAction'...
[move_group-1] Loading 'move_group/MoveGroupGetPlanningSceneService'...
[move_group-1] Loading 'move_group/MoveGroupKinematicsService'...
[move_group-1] Loading 'move_group/MoveGroupMoveAction'...
[move_group-1] Loading 'move_group/MoveGroupPlanService'...
[move_group-1] Loading 'move_group/MoveGroupQueryPlannersService'...
[move_group-1] Loading 'move_group/MoveGroupStateValidationService'...
[move_group-1]
[move_group-1] You can start planning now!
[move_group-1]
[rviz2-3] [INFO] [1633437090.833687600] [rviz2]: Stereo is NOT SUPPORTED
[rviz2-3] [INFO] [1633437090.833839700] [rviz2]: OpenGl version: 3.1 (GLSL 1.4)
[rviz2-3] [INFO] [1633437090.870951800] [rviz2]: Stereo is NOT SUPPORTED
[mongo_wrapper_ros.py-2] [INFO] [1633437090.982140000] [mongodb]: Starting mongodb with db location /tmp/db listening on localhost:33829
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.012+0200 I CONTROL  [initandlisten] MongoDB starting : pid=11507 port=33829 dbpath=/tmp/db 64-bit host=NBK-18
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.012+0200 I CONTROL  [initandlisten] db version v3.6.8
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.012+0200 I CONTROL  [initandlisten] git version: 8e540c0b6db93ce994cc548f000900bdc740f80a
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.012+0200 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.1.1f  31 Mar 2020
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.012+0200 I CONTROL  [initandlisten] allocator: tcmalloc
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.012+0200 I CONTROL  [initandlisten] modules: none
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.012+0200 I CONTROL  [initandlisten] build environment:
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.012+0200 I CONTROL  [initandlisten]     distarch: x86_64
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.012+0200 I CONTROL  [initandlisten]     target_arch: x86_64
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.012+0200 I CONTROL  [initandlisten] options: { net: { port: 33829 }, storage: { dbPath: "/tmp/db" } }
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.012+0200 I -        [initandlisten] Detected data files in /tmp/db created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.012+0200 I STORAGE  [initandlisten]
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.012+0200 I STORAGE  [initandlisten] ** WARNING: Using the XFS filesystem is strongly recommended with the WiredTiger storage engine
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.012+0200 I STORAGE  [initandlisten] **          See http://dochub.mongodb.org/core/prodnotes-filesystem
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.012+0200 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=12286M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),cache_cursors=false,compatibility=(release="3.0",require_max="3.0"),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
[rviz2-3] Warning: class_loader.impl: SEVERE WARNING!!! A namespace collision has occured with plugin factory for class rviz_default_plugins::displays::InteractiveMarkerDisplay. New factory will OVERWRITE existing one. This situation occurs when libraries containing plugins are directly linked against an executable (the one running right now generating this message). Please separate plugins out into their own library or just don't link against the library and use either class_loader::ClassLoader/MultiLibraryClassLoader to open.
[rviz2-3]          at line 253 in /opt/ros/foxy/include/class_loader/class_loader_core.hpp
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.752+0200 I STORAGE  [initandlisten] WiredTiger message [1633437091:752876][11507:0x7fd4dc17ff40], txn-recover: Main recovery loop: starting at 92/5120
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.833+0200 I STORAGE  [initandlisten] WiredTiger message [1633437091:833784][11507:0x7fd4dc17ff40], txn-recover: Recovering log 92 through 93
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.896+0200 I STORAGE  [initandlisten] WiredTiger message [1633437091:896833][11507:0x7fd4dc17ff40], txn-recover: Recovering log 93 through 93
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.936+0200 I STORAGE  [initandlisten] WiredTiger message [1633437091:936581][11507:0x7fd4dc17ff40], txn-recover: Set global recovery timestamp: 0
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.984+0200 I CONTROL  [initandlisten]
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.984+0200 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.984+0200 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.984+0200 I CONTROL  [initandlisten]
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.984+0200 I CONTROL  [initandlisten] ** WARNING: This server is bound to localhost.
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.984+0200 I CONTROL  [initandlisten] **          Remote systems will be unable to connect to this server.
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.985+0200 I CONTROL  [initandlisten] **          Start the server with --bind_ip <address> to specify which IP
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.985+0200 I CONTROL  [initandlisten] **          addresses it should serve responses from, or with --bind_ip_all to
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.985+0200 I CONTROL  [initandlisten] **          bind to all interfaces. If this behavior is desired, start the
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.985+0200 I CONTROL  [initandlisten] **          server with --bind_ip 127.0.0.1 to disable this warning.
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.985+0200 I CONTROL  [initandlisten]
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.985+0200 I CONTROL  [initandlisten]
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.985+0200 I CONTROL  [initandlisten] ** WARNING: /sys/kernel/mm/transparent_hugepage/enabled is 'always'.
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.985+0200 I CONTROL  [initandlisten] **        We suggest setting it to 'never'
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:31.985+0200 I CONTROL  [initandlisten]
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:32.038+0200 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/tmp/db/diagnostic.data'
[mongo_wrapper_ros.py-2] 2021-10-05T14:31:32.039+0200 I NETWORK  [initandlisten] waiting for connections on port 33829
[rviz2-3] [ERROR] [1633437094.179407000] [moveit_ros_visualization.motion_planning_frame]: Action server: /recognize_objects not available
[rviz2-3] Parsing robot urdf xml string.
[rviz2-3] [INFO] [1633437094.305097100] [moveit_rdf_loader.rdf_loader]: Loaded robot model in 0.0034292 seconds
[rviz2-3] [INFO] [1633437094.305303200] [moveit_robot_model.robot_model]: Loading robot model 'ur'...
[rviz2-3] Link base had 0 children
[rviz2-3] Link base_link_inertia had 1 children
[rviz2-3] Link shoulder_link had 1 children
[rviz2-3] Link upper_arm_link had 1 children
[rviz2-3] Link forearm_link had 1 children
[rviz2-3] Link wrist_1_link had 1 children
[rviz2-3] Link wrist_2_link had 1 children
[rviz2-3] Link wrist_3_link had 1 children
[rviz2-3] Link flange had 1 children
[rviz2-3] Link tool0 had 0 children
[rviz2-3] [INFO] [1633437094.469863900] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Starting planning scene monitor
[rviz2-3] [INFO] [1633437094.471826000] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Listening to '/monitored_planning_scene'
[rviz2-3] [INFO] [1633437095.687823600] [interactive_marker_display_93861557128848]: Connected on namespace: rviz_moveit_motion_planning_display/robot_interaction_interactive_marker_topic
[rviz2-3] Link base had 0 children
[rviz2-3] Link base_link_inertia had 1 children
[rviz2-3] Link shoulder_link had 1 children
[rviz2-3] Link upper_arm_link had 1 children
[rviz2-3] Link forearm_link had 1 children
[rviz2-3] Link wrist_1_link had 1 children
[rviz2-3] Link wrist_2_link had 1 children
[rviz2-3] Link wrist_3_link had 1 children
[rviz2-3] Link flange had 1 children
[rviz2-3] Link tool0 had 0 children
[rviz2-3] [INFO] [1633437095.714628800] [moveit_ros_visualization.motion_planning_frame]: group ur_manipulator
[rviz2-3] [INFO] [1633437095.714747800] [moveit_ros_visualization.motion_planning_frame]: Constructing new MoveGroup connection for group 'ur_manipulator' in namespace ''
[rviz2-3] [WARN] [1633437095.714958400] [rcl.logging_rosout]: Publisher already registered for provided node name. If this is due to multiple nodes with the same name then all logs for that logger name will go out over the existing publisher. As soon as any node with that name is destructed it will unregister the publisher, preventing any further logs for that name from being published on the rosout topic.
[rviz2-3] [INFO] [1633437095.759271300] [move_group_interface]: Ready to take commands for planning group ur_manipulator.
[rviz2-3] [INFO] [1633437095.759380700] [move_group_interface]: Looking around: no
[rviz2-3] [INFO] [1633437095.759419300] [move_group_interface]: Replanning: no
[rviz2-3] [INFO] [1633437095.862541000] [interactive_marker_display_93861557128848]: Sending request for interactive markers
[rviz2-3] [INFO] [1633437095.870286200] [moveit_ros_visualization.motion_planning_frame_planning]: POPULATING PLANNERS 12 grp: ur_manipulator
[rviz2-3] [INFO] [1633437095.870578800] [moveit_ros_visualization.motion_planning_frame_planning]: planner id: ur_manipulator
[rviz2-3] [INFO] [1633437095.870598200] [moveit_ros_visualization.motion_planning_frame_planning]: planner id: ur_manipulator[BKPIECEkConfigDefault]
[rviz2-3] [INFO] [1633437095.870734100] [moveit_ros_visualization.motion_planning_frame_planning]: planner id: ur_manipulator[ESTkConfigDefault]
[rviz2-3] [INFO] [1633437095.870743600] [moveit_ros_visualization.motion_planning_frame_planning]: planner id: ur_manipulator[KPIECEkConfigDefault]
[rviz2-3] [INFO] [1633437095.870748600] [moveit_ros_visualization.motion_planning_frame_planning]: planner id: ur_manipulator[LBKPIECEkConfigDefault]
[rviz2-3] [INFO] [1633437095.870753200] [moveit_ros_visualization.motion_planning_frame_planning]: planner id: ur_manipulator[PRMkConfigDefault]
[rviz2-3] [INFO] [1633437095.870757300] [moveit_ros_visualization.motion_planning_frame_planning]: planner id: ur_manipulator[PRMstarkConfigDefault]
[rviz2-3] [INFO] [1633437095.870762100] [moveit_ros_visualization.motion_planning_frame_planning]: planner id: ur_manipulator[RRTConnectkConfigDefault]
[rviz2-3] [INFO] [1633437095.870766500] [moveit_ros_visualization.motion_planning_frame_planning]: planner id: ur_manipulator[RRTkConfigDefault]
[rviz2-3] [INFO] [1633437095.870770800] [moveit_ros_visualization.motion_planning_frame_planning]: planner id: ur_manipulator[RRTstarkConfigDefault]
[rviz2-3] [INFO] [1633437095.870775200] [moveit_ros_visualization.motion_planning_frame_planning]: planner id: ur_manipulator[SBLkConfigDefault]
[rviz2-3] [INFO] [1633437095.870779400] [moveit_ros_visualization.motion_planning_frame_planning]: planner id: ur_manipulator[TRRTkConfigDefault]
[rviz2-3] [INFO] [1633437095.940072200] [interactive_marker_display_93861557128848]: Service response received for initialization
[move_group-1] [INFO] [1633437162.465320700] [moveit_move_group_default_capabilities.move_action_capability]: Received request
[move_group-1] [INFO] [1633437162.465670700] [moveit_move_group_default_capabilities.move_action_capability]: executing..
[move_group-1] [INFO] [1633437162.466760500] [moveit_move_group_default_capabilities.move_action_capability]: Planning request received for MoveGroup action. Forwarding to planning pipeline.
[move_group-1] [INFO] [1633437162.466896700] [moveit_move_group_capabilities_base.move_group_capability]: Using planning pipeline 'move_group'
[move_group-1] [INFO] [1633437162.468886100] [moveit.ompl_planning.model_based_planning_context]: Planner configuration 'ur_manipulator[RRTConnectkConfigDefault]' will use planner 'geometric::RRTConnect'. Additional configuration parameters will be set when the planner is constructed.
[move_group-1] [INFO] [1633437162.470676500] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/geometric/planners/rrt/src/RRTConnect.cpp:220 - ur_manipulator/ur_manipulator[RRTConnectkConfigDefault]: Starting planning with 1 states already in datastructure
[move_group-1] [INFO] [1633437162.470855000] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/geometric/planners/rrt/src/RRTConnect.cpp:220 - ur_manipulator/ur_manipulator[RRTConnectkConfigDefault]: Starting planning with 1 states already in datastructure
[move_group-1] [INFO] [1633437162.470928500] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/geometric/planners/rrt/src/RRTConnect.cpp:220 - ur_manipulator/ur_manipulator[RRTConnectkConfigDefault]: Starting planning with 1 states already in datastructure
[move_group-1] [INFO] [1633437162.470964600] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/geometric/planners/rrt/src/RRTConnect.cpp:220 - ur_manipulator/ur_manipulator[RRTConnectkConfigDefault]: Starting planning with 1 states already in datastructure
[move_group-1] [INFO] [1633437162.482049400] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/geometric/planners/rrt/src/RRTConnect.cpp:354 - ur_manipulator/ur_manipulator[RRTConnectkConfigDefault]: Created 5 states (3 start + 2 goal)
[move_group-1] [INFO] [1633437162.482638000] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/geometric/planners/rrt/src/RRTConnect.cpp:354 - ur_manipulator/ur_manipulator[RRTConnectkConfigDefault]: Created 5 states (3 start + 2 goal)
[move_group-1] [INFO] [1633437162.482763000] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/geometric/planners/rrt/src/RRTConnect.cpp:354 - ur_manipulator/ur_manipulator[RRTConnectkConfigDefault]: Created 5 states (3 start + 2 goal)
[move_group-1] [INFO] [1633437162.483399400] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/geometric/planners/rrt/src/RRTConnect.cpp:354 - ur_manipulator/ur_manipulator[RRTConnectkConfigDefault]: Created 4 states (2 start + 2 goal)
[move_group-1] [INFO] [1633437162.483604000] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/tools/multiplan/src/ParallelPlan.cpp:135 - ParallelPlan::solve(): Solution found by one or more threads in 0.014189 seconds
[move_group-1] [INFO] [1633437162.483902000] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/geometric/planners/rrt/src/RRTConnect.cpp:220 - ur_manipulator/ur_manipulator[RRTConnectkConfigDefault]: Starting planning with 1 states already in datastructure
[move_group-1] [INFO] [1633437162.484012400] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/geometric/planners/rrt/src/RRTConnect.cpp:220 - ur_manipulator/ur_manipulator[RRTConnectkConfigDefault]: Starting planning with 1 states already in datastructure
[move_group-1] [INFO] [1633437162.484104100] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/geometric/planners/rrt/src/RRTConnect.cpp:220 - ur_manipulator/ur_manipulator[RRTConnectkConfigDefault]: Starting planning with 1 states already in datastructure
[move_group-1] [INFO] [1633437162.484131300] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/geometric/planners/rrt/src/RRTConnect.cpp:220 - ur_manipulator/ur_manipulator[RRTConnectkConfigDefault]: Starting planning with 1 states already in datastructure
[move_group-1] [INFO] [1633437162.484911200] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/geometric/planners/rrt/src/RRTConnect.cpp:354 - ur_manipulator/ur_manipulator[RRTConnectkConfigDefault]: Created 4 states (2 start + 2 goal)
[move_group-1] [INFO] [1633437162.484959800] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/geometric/planners/rrt/src/RRTConnect.cpp:354 - ur_manipulator/ur_manipulator[RRTConnectkConfigDefault]: Created 4 states (2 start + 2 goal)
[move_group-1] [INFO] [1633437162.485927800] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/geometric/planners/rrt/src/RRTConnect.cpp:354 - ur_manipulator/ur_manipulator[RRTConnectkConfigDefault]: Created 4 states (2 start + 2 goal)
[move_group-1] [INFO] [1633437162.486215300] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/geometric/planners/rrt/src/RRTConnect.cpp:354 - ur_manipulator/ur_manipulator[RRTConnectkConfigDefault]: Created 5 states (2 start + 3 goal)
[move_group-1] [INFO] [1633437162.486358500] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/tools/multiplan/src/ParallelPlan.cpp:135 - ParallelPlan::solve(): Solution found by one or more threads in 0.002578 seconds
[move_group-1] [INFO] [1633437162.486602700] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/geometric/planners/rrt/src/RRTConnect.cpp:220 - ur_manipulator/ur_manipulator[RRTConnectkConfigDefault]: Starting planning with 1 states already in datastructure
[move_group-1] [INFO] [1633437162.486721100] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/geometric/planners/rrt/src/RRTConnect.cpp:220 - ur_manipulator/ur_manipulator[RRTConnectkConfigDefault]: Starting planning with 1 states already in datastructure
[move_group-1] [INFO] [1633437162.488282000] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/geometric/planners/rrt/src/RRTConnect.cpp:354 - ur_manipulator/ur_manipulator[RRTConnectkConfigDefault]: Created 5 states (2 start + 3 goal)
[move_group-1] [INFO] [1633437162.490947100] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/geometric/planners/rrt/src/RRTConnect.cpp:354 - ur_manipulator/ur_manipulator[RRTConnectkConfigDefault]: Created 5 states (2 start + 3 goal)
[move_group-1] [INFO] [1633437162.491125500] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/tools/multiplan/src/ParallelPlan.cpp:135 - ParallelPlan::solve(): Solution found by one or more threads in 0.004634 seconds
[move_group-1] [INFO] [1633437162.493437400] [ompl]: /tmp/binarydeb/ros-foxy-ompl-1.5.0/src/ompl/geometric/src/SimpleSetup.cpp:179 - SimpleSetup: Path simplification took 0.002171 seconds and changed from 3 to 2 states
[move_group-1] [INFO] [1633437162.500671200] [moveit_move_group_default_capabilities.move_action_capability]: Motion plan was computed succesfully.
[rviz2-3] [INFO] [1633437162.866289300] [move_group_interface]: Planning request accepted
[rviz2-3] [INFO] [1633437162.966858300] [move_group_interface]: Planning request complete!
[rviz2-3] [INFO] [1633437163.067073000] [move_group_interface]: time taken to generate plan: 0.0240412 seconds
[move_group-1] [INFO] [1633437212.704172100] [moveit_move_group_default_capabilities.move_action_capability]: Received request
[move_group-1] [INFO] [1633437212.704324800] [moveit_move_group_default_capabilities.move_action_capability]: executing..
[move_group-1] [INFO] [1633437212.705621100] [moveit_move_group_default_capabilities.move_action_capability]: Planning request received for MoveGroup action. Forwarding to planning pipeline.
```

--- a/ur_bringup/launch/ur_moveit.launch.py
+++ b/ur_bringup/launch/ur_moveit.launch.py
@@ -179,15 +179,6 @@ def launch_setup(context, *args, **kwargs):
         "publish_geometry_updates": True,
         "publish_state_updates": True,
         "publish_transforms_updates": True,
-        "planning_scene_monitor_options": {
-            "name": "planning_scene_monitor",
-            "robot_description": "robot_description",
-            "joint_state_topic": "/joint_states",
-            "attached_collision_object_topic": "/move_group/planning_scene_monitor",
-            "publish_planning_scene_topic": "/move_group/publish_planning_scene",
-            "monitored_planning_scene_topic": "/move_group/monitored_planning_scene",
-            "wait_for_initial_state_timeout": 10.0,
-        },
     }
 
     # Start the actual move_group node/action server

